client/src/App.jsx

```diff
@@ -25,7 +23,7 @@ const App = ({ history, location }) => (
 );
 
 App.propTypes = {
-  // history: PropTypes.shape().isRequired,
-  // location: PropTypes.shape().isRequired,
+  history: PropTypes.shape().isRequired,
+  location: PropTypes.shape().isRequired,
```

Comment:
Since `shape()` is empty it's not really checking for any particular shape and partially defeats the purpose of `propTypes`.

Reply:
Defined the explicit shape